android/app/src/main/java/io/github/parkwithease/parkeasy/HttpService.kt

```diff
@@ -0,0 +1,51 @@
+package io.github.parkwithease.parkeasy
+
+import android.util.Log
+import io.ktor.client.HttpClient
+import io.ktor.client.engine.android.Android
+import io.ktor.client.plugins.contentnegotiation.ContentNegotiation
+import io.ktor.client.plugins.cookies.HttpCookies
+import io.ktor.client.plugins.logging.LogLevel
+import io.ktor.client.plugins.logging.Logger
+import io.ktor.client.plugins.logging.Logging
+import io.ktor.client.request.post
+import io.ktor.client.request.setBody
+import io.ktor.http.ContentType
+import io.ktor.http.Cookie
+import io.ktor.http.contentType
+import io.ktor.http.setCookie
+import io.ktor.serialization.kotlinx.json.json
+
+object HttpService {
+    const val API_HOST = "http://10.0.2.2:8080"
```

Comment:
Can we make another file to store these constant?

Reply:
And also, just curios why the IP here is 10.0.2.2 (it works but... just curious)

Reply:
> And also, just curios why the IP here is 10.0.2.2 (it works but... just curious)

It's referencing 127.0.0.1 on the actual computer running the emulator
https://developer.android.com/studio/run/emulator-networking

Reply:
API_HOST string moved into strings.xml